sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_connections_operations.py

```diff
@@ -138,8 +143,10 @@ def delete(self, name: str) -> None:
             connection_name=name,
             workspace_name=self._workspace_name,
             **self._scope_kwargs,
+            **kwargs,
```

Comment:
Is there any possibility that there could be parameter name conflicts between `scope_kwargs` and `kwargs`? The possible supported parameter names for kwargs can be found here (in the parameters column):
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#available-policies